## Exercício 1: Vestibular

Considere que a os dados gerados na célula abaixo contêm o número de acertos de 100 alunos em um vestibular para um curso de exatas, divididas pelos respectivos assuntos. Considere que cada assunto possui um número de questões conforme a tabela abaixo:

| assunto | número de questões |
|:---:|:---:|
| Matemática | 24 |
| Português | 18 |
| Geografia | 8 |
| Inglês | 8 |
| História | 8 |
| Física | 12 |
| Química | 12 |

Usando os comandos de operações com DataFrames que você aprendeu na Aula 03, calcule:

1. (operações com escalar) Calcule o percentual de acerto dos alunos por assunto.  
2. (operações entre *DataFrames) Calcule o total de acertos de cada aluno.  
3. Calcule o porcentual geral de cada aluno.  
4. Suponha que a nota de corte para a segunda fase seja 45. Quantos alunos tiveram nota maior que 45?  

In [28]:
import pandas as pd
import numpy as np
np.random.seed(42)
df_mat = pd.DataFrame(np.random.randint(24, size = (100,1)), columns = ['Qt de acertos'])
df_por = pd.DataFrame(np.random.randint(18, size = (100,1)), columns = ['Qt de acertos'])
df_geo = pd.DataFrame(np.random.randint(8, size = (100,1)), columns = ['Qt de acertos'])
df_ing = pd.DataFrame(np.random.randint(8, size = (100,1)), columns = ['Qt de acertos'])
df_his = pd.DataFrame(np.random.randint(8, size = (100,1)), columns = ['Qt de acertos'])
df_fis = pd.DataFrame(np.random.randint(12, size = (100,1)), columns = ['Qt de acertos'])
df_qui = pd.DataFrame(np.random.randint(12, size = (100,1)), columns = ['Qt de acertos'])

Qt de acertos
0               6
1              19
2              14
3              10
4               7
..            ...
95              6
96              8
97             23
98              0
99             11

[100 rows x 1 columns]

In [26]:
# 1) Seu código aqui
por_mat = df_mat/24
por_por = df_por/18
por_geo = df_geo/8
por_ing = df_ing/8
por_his = df_his/8
por_fis = df_fis/12
por_qui = df_qui/12


In [27]:
# 2) Seu código aqui
df_tottal = df_mat + df_por + df_geo + df_ing + df_his + df_fis + df_qui
df_tottal

Qt de acertos
0              35
1              48
2              50
3              32
4              30
..            ...
95             34
96             27
97             40
98             40
99             42

[100 rows x 1 columns]

In [29]:
# 3) Seu código aqui
por_df_total = df_tottal/90
por_df_total

Qt de acertos
0        0.388889
1        0.533333
2        0.555556
3        0.355556
4        0.333333
..            ...
95       0.377778
96       0.300000
97       0.444444
98       0.444444
99       0.466667

[100 rows x 1 columns]

In [38]:
# 4) Seu código aqui
corte = df_tottal[df_tottal['Qt de acertos'] > 45]
corte

Qt de acertos
1              48
2              50
5              50
7              49
8              58
17             52
18             50
20             53
24             50
27             50
31             54
35             48
36             56
37             52
39             49
40             47
42             48
45             48
48             55
60             48
62             46
67             46
70             62
71             53
72             50
77             53
78             66
81             55
88             50
89             63
93             56

In [39]:
corte.shape

(31, 1)

31 alunos tiveram notas maiores que 45.

## 2) Vestibular II

Ainda sobre o mesmo banco de dados:

1. Neste vestibular, quem 'zera' em matemática, física ou química está desqualificado. Monte um novo *DataFrame* com os alunos desqualificados por este critério.
2. Quantos são esses alunos?
3. Qual a média desses alunos em história e geografia?
4. Monte um *DataFrame* com os alunos que passaram para a segunda fase. Repare que estes alunos não podem ter sido desqualificados.

In [36]:
# seu código aqui
mat = df_mat['Qt de acertos']
por = df_por['Qt de acertos']
geo = df_geo['Qt de acertos']
ing = df_ing['Qt de acertos']
his = df_his['Qt de acertos']
fis = df_fis['Qt de acertos']
qui = df_qui['Qt de acertos']
Tabela_Acertos =pd.DataFrame({'Matemática': mat, 'Português': por, 'Geografia': geo, 'Inglês': ing, 'História': his, 'Física': 
                             fis, 'Química': qui}) 
Tabela_Acertos

Matemática  Português  Geografia  Inglês  História  Física  Química
0            6          7          7       3         1       8        3
1           19         10          0       0         0      11        8
2           14         16          0       5         7       4        4
3           10          7          3       4         0       0        8
4            7          2          2       7         0       9        3
..         ...        ...        ...     ...       ...     ...      ...
95           6          4          7       1         4       9        3
96           8          1          4       6         1       4        3
97          23          5          0       2         4       1        5
98           0         10          3       1         5      10       11
99          11         15          0       1         6       7        2

[100 rows x 7 columns]

In [45]:
Desqualificados = Tabela_Acertos[(Tabela_Acertos['Matemática'] == 0) | (Tabela_Acertos['Química'] == 0) | (Tabela_Acertos['Física'] == 0)]
Desqualificados

Matemática  Português  Geografia  Inglês  História  Física  Química
3           10          7          3       4         0       0        8
6            6          0          7       7         5       0        4
10          10          8          3       2         1       7        0
12          20          8          3       7         1       0        3
14           7         11          1       7         1       4        0
20          23         11          6       5         6       2        0
25           0          4          7       4         0       4        8
26          11         14          6       2         7       0        0
27          21         13          1       5         7       3        0
33          14          8          0       5         6       6        0
35          11         14          6       3         3       0       11
49           8          1          1       0         1       0        8
56           7          6          7       5         6       0        8
65           1          8          2       4         3       0        6
66           5          4          3       6         3       0       10
72          11         16          3       4         7       0        9
73           1         16          2       4         3       0       11
74           9         11          0       7         5       7        0
84          15          1          0       5         1       0        4
89          23         11          7       7         6       9        0
92          14         10          1       6         1       8        0
94           0          5          2       6         1       3        8
98           0         10          3       1         5      10       11

In [46]:
Desqualificados.shape

(23, 7)

23 candidatos foram desqualificados

## 3) Vacinações no Acre
Vamos trabalhar agora com a base de vacinações no Acre. Para facilitar a sua vida, copiamos o link do arquivo na célula abaixo.

1. Quantas vacinas estão registradas nessa base?  
2. Quantos pacientes foram vacinados? (considere um paciente para cada valor único de ```paciente_id```)  
3. Quantos pacientes únicos tomaram a primeira dose? OBS: Há um caractere especial neste campo. Receba os valores do campo com o método ```.unique()```.   
4. Quantos pacientes com menos de 18 anos foram vacinados?  
5. Quantos estabelecimentos aplicaram vacina no Acre?


**OBS:** O portal do DATASUS pode apresentar instabilidades, retornando um erro na segunda célula abaixo. Por este motivo está disponível uma base estática, que se for baixada para o seu *working directory* pode ser lida com este comando: ```df = pd.read_csv('registros de vacinacao covid ACRE.csv', sep=';')```.

**OBS2:** Para saber qual é o seu working directory, rode no jupyter: ```!pwd```.

In [60]:
# 1) Sua solução aqui
vacinas = pd.read_csv("part-00000-03d822a4-2f78-4ea4-832a-5c531c0a0715.c000.csv" , sep= ';')
vacinas

C:\Users\ale-s\AppData\Local\Temp\ipykernel_25152\435279977.py:2: DtypeWarning: Columns (7,8) have mixed types. Specify dtype option on import or set low_memory=False.
  vacinas = pd.read_csv("part-00000-03d822a4-2f78-4ea4-832a-5c531c0a0715.c000.csv" , sep= ';')


document_id  \
0       d2733f08-a5a1-41fe-ae24-fed03778e469-i0b0   
1       f9977bcd-8d71-445e-bd89-e04c4f1de8b7-i0b0   
2       5ad43e8f-e740-4356-bb6d-6b1ea56c2ab0-i0b0   
3       f9b763e1-1fe4-4a02-808f-371c04d2eb24-i0b0   
4       ebdeedec-1323-4925-b32a-66fb64c06b8e-i0b0   
...                                           ...   
545075  c850eac2-0d6c-476a-862f-98b536abc5dc-i0b0   
545076  c8554062-2bce-4ccf-bfd3-ad8f45ef9e40-i0b0   
545077  c873ae38-4533-44cc-a5cd-e297c996e84d-i0b0   
545078  c8846ff8-36fe-42bf-83ce-ab63b5559de3-i0b0   
545079  c8a54259-cbc0-4520-90bd-3757baf8bc39-i0b0   

                                              paciente_id  paciente_idade  \
0       d019364b6c8b5ffe2693648a23ad3a2ec9dae72ac0d095...            61.0   
1       ecd75b63c7a227c98b7d7ad3af6beb36aaad232b960b31...            40.0   
2       4f51e4b4126e6e47acd7a551240a8ea3980dfec13ae4d5...            28.0   
3       627974108a80f1129654913f412df03136ed23a1c719a3...            18.0   
4       189d28e54eede70f6034cb7664ba2703c6776762272d23...            81.0   
...                                                   ...             ...   
545075  43b21abb2fee080cefeb6409b4f761ba5c16687030c200...            26.0   
545076  fa37dd0d287afe8d39054a911fdd1cbdd51f5ba3af5aa3...            58.0   
545077  cdc18f17dc86770e9d7c681fa4d0b31894ee041ec77160...            66.0   
545078  0c2083a9c52d580e839beb7d8e3d878b08b284c6841976...            78.0   
545079  eae350ea4195f35bc6d5a648ab084faa41646991d0ccfa...            17.0   

       paciente_dataNascimento paciente_enumSexoBiologico  \
0                   1960-05-28                          F   
1                   1981-10-05                          F   
2                   1993-06-17                          F   
3                   2003-01-19                          F   
4                   1939-12-11                          F   
...                        ...                        ...   
545075              1995-09-30                          F   
545076              1963-09-04                          F   
545077              1954-06-16                          F   
545078              1943-02-18                          F   
545079              2004-04-28                          F   

        paciente_racaCor_codigo paciente_racaCor_valor  \
0                           3.0                  PARDA   
1                           4.0                AMARELA   
2                           3.0                  PARDA   
3                           1.0                 BRANCA   
4                           4.0                AMARELA   
...                         ...                    ...   
545075                      4.0                AMARELA   
545076                      2.0                  PRETA   
545077                      4.0                AMARELA   
545078                      3.0                  PARDA   
545079                      4.0                AMARELA   

       paciente_endereco_coIbgeMunicipio paciente_endereco_coPais  \
0                               120030.0                     10.0   
1                               120050.0                     10.0   
2                               120020.0                     10.0   
3                               120040.0                     10.0   
4                               120050.0                     10.0   
...                                  ...                      ...   
545075                            520870                       10   
545076                            120040                       10   
545077                            120050                       10   
545078                            130260                       10   
545079                            120030                       10   

       paciente_endereco_nmMunicipio  ... vacina_categoria_codigo  \
0                              FEIJO  ...                     2.0   
1                     SENA MADUREIRA  ...                     2.0   
2         

In [61]:
vacinas.shape

(545080, 32)

Estão registradas 545080 vacinas


In [63]:
vacinas.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 545080 entries, 0 to 545079
Data columns (total 32 columns):
 #   Column                                    Non-Null Count   Dtype  
---  ------                                    --------------   -----  
 0   document_id                               545080 non-null  object 
 1   paciente_id                               545079 non-null  object 
 2   paciente_idade                            545079 non-null  float64
 3   paciente_dataNascimento                   545079 non-null  object 
 4   paciente_enumSexoBiologico                545079 non-null  object 
 5   paciente_racaCor_codigo                   545079 non-null  float64
 6   paciente_racaCor_valor                    545079 non-null  object 
 7   paciente_endereco_coIbgeMunicipio         541790 non-null  object 
 8   paciente_endereco_coPais                  541797 non-null  object 
 9   paciente_endereco_nmMunicipio             541796 non-null  object 
 10  paciente_endereco_nm

In [71]:
# 2) 
vacinas['paciente_id'].value_counts

<bound method IndexOpsMixin.value_counts of 0         d019364b6c8b5ffe2693648a23ad3a2ec9dae72ac0d095...
1         ecd75b63c7a227c98b7d7ad3af6beb36aaad232b960b31...
2         4f51e4b4126e6e47acd7a551240a8ea3980dfec13ae4d5...
3         627974108a80f1129654913f412df03136ed23a1c719a3...
4         189d28e54eede70f6034cb7664ba2703c6776762272d23...
                                ...                        
545075    43b21abb2fee080cefeb6409b4f761ba5c16687030c200...
545076    fa37dd0d287afe8d39054a911fdd1cbdd51f5ba3af5aa3...
545077    cdc18f17dc86770e9d7c681fa4d0b31894ee041ec77160...
545078    0c2083a9c52d580e839beb7d8e3d878b08b284c6841976...
545079    eae350ea4195f35bc6d5a648ab084faa41646991d0ccfa...
Name: paciente_id, Length: 545080, dtype: object>

In [79]:
# 3) solução alternativa
vacinas['vacina_descricao_dose'] == '1ª Dose'

0          True
1         False
2         False
3          True
4         False
          ...  
545075    False
545076    False
545077     True
545078     True
545079     True
Name: vacina_descricao_dose, Length: 545080, dtype: bool

In [85]:
vacinas[vacinas['vacina_descricao_dose'] == '1ª Dose'].shape

(227382, 32)

In [ ]:
227382 tomaram a primeira dose

In [88]:
# 4) 
vacinas[vacinas['paciente_idade']<18].shape

(89769, 32)

In [98]:
# 5)
len(vacinas['estabelecimento_valor'].unique())

144

144 estabelecimentos aplicam vacinas

## 4) Vacinação II
Gere um *DataFrame* que contenha somente os estabelecimentos que aplicaram vcinas a menores de 18 anos. Nesse *DataFrame* devem conter somente os dados dos estabelecimentos, mais uma coluna sendo a quantidade de vacinas que o estabelecimento aplicou a menores de 18 anos.  
  
1. crie uma cópia do *DataFrame* original, contendo somente os registros de vacinas realizadas a menores de 18 anos.  
2. crie uma lista das colunas desse *DataFrame* com o atributo de *DataFrame* **.columns()**  
3. Nesse *DataFrame* faça uma contagem do campo ```vacina_categoria_nome```.
3. a partir da lista de colunas, escolha somente aquelas que são referentes ao estabelecimento, faça uma lista com esses valores.  
4. usando o método *.loc*, selecione somente essas variáveis  
5. Aplique o método **.drop_duplicates** e crie uma lista com uma linha para cada estabelecimento, com os dados do estabelecimento  

In [102]:
# 1) 
vacinas_copy = vacinas[vacinas['paciente_idade']<18].copy()
vacinas_copy

document_id  \
13      fa575a1f-8608-45e4-9a74-04aff6f51eb7-i0b0   
15      fa88c7ab-2dfe-41d3-b023-4162734ebf38-i0b0   
22      329a5d1a-9194-4208-8bdc-dacdc371a180-i0b0   
33      fc0dfc6e-3c57-4e25-98ad-b4f030938922-i0b0   
38      ca02a9e7-6ace-4e55-97ef-d87e251f1724-i0b0   
...                                           ...   
545048  c4eba952-c5e4-4c95-8e5f-0b2f0127ac35-i0b0   
545052  c51f73ba-3bb6-4a5e-9163-ce590892ff7a-i0b0   
545057  c5e115f6-33b7-4011-90f5-c9bcdd6d7ee5-i0b0   
545071  c72c8b5a-2b22-4138-bc16-1e6cfa7e8be3-i0b0   
545079  c8a54259-cbc0-4520-90bd-3757baf8bc39-i0b0   

                                              paciente_id  paciente_idade  \
13      c77f4461957d0146922cc801ed80c34c336e9058523c0e...            13.0   
15      02bc0b701a9094b3951490aa227f538d6abaedfd6596cb...            12.0   
22      8902af8a95706c8a4f1de4222ba274790f1fb281632b84...            13.0   
33      1d9abab4c1580111247e25fab6ca6d8d282bdd3ae15ea9...             8.0   
38      f9e431adcffad5e3944912ed2b9fe3f385846fa53e5556...             8.0   
...                                                   ...             ...   
545048  ca891dd8f590ac5e08e3775551ca64ade805bd0bcbe2b6...            14.0   
545052  899d7384b1ce6df38bb337010d882afd1e7a4612e4dd3b...             7.0   
545057  5d5573306d7ea9ef4a0e3020dcc67d9febb4eeefdf9dd8...            13.0   
545071  2ef7f0e93fedbcd80b062426fb581d90624649bfdbe90f...            15.0   
545079  eae350ea4195f35bc6d5a648ab084faa41646991d0ccfa...            17.0   

       paciente_dataNascimento paciente_enumSexoBiologico  \
13                  2008-08-08                          F   
15                  2009-04-24                          M   
22                  2008-09-02                          F   
33                  2013-10-01                          F   
38                  2013-09-18                          M   
...                        ...                        ...   
545048              2006-12-22                          F   
545052              2014-06-29                          F   
545057              2008-08-10                          F   
545071              2006-12-15                          M   
545079              2004-04-28                          F   

        paciente_racaCor_codigo paciente_racaCor_valor  \
13                          3.0                  PARDA   
15                          3.0                  PARDA   
22                         99.0         SEM INFORMACAO   
33                          3.0                  PARDA   
38                          3.0                  PARDA   
...                         ...                    ...   
545048                      1.0                 BRANCA   
545052                      5.0               INDIGENA   
545057                      3.0                  PARDA   
545071                      2.0                  PRETA   
545079                      4.0                AMARELA   

       paciente_endereco_coIbgeMunicipio paciente_endereco_coPais  \
13                              120020.0                     10.0   
15                              120040.0                     10.0   
22                              120040.0                     10.0   
33                              120034.0                     10.0   
38                              120040.0                     10.0   
...                                  ...                      ...   
545048                            520870                       10   
545052                            120033                       10   
545057                            120040                       10   
545071                            120032                       10   
545079                            120030                       10   

       paciente_endereco_nmMunicipio  ... vacina_categoria_codigo  \
13                   CRUZEIRO DO SUL  ...                   114.0   
15                        RIO BRANCO  ...                     2.0   
22        

In [110]:
# 2) 
vacinas_copy.columns

Index(['document_id', 'paciente_id', 'paciente_idade',
       'paciente_dataNascimento', 'paciente_enumSexoBiologico',
       'paciente_racaCor_codigo', 'paciente_racaCor_valor',
       'paciente_endereco_coIbgeMunicipio', 'paciente_endereco_coPais',
       'paciente_endereco_nmMunicipio', 'paciente_endereco_nmPais',
       'paciente_endereco_uf', 'paciente_endereco_cep',
       'paciente_nacionalidade_enumNacionalidade', 'estabelecimento_valor',
       'estabelecimento_razaoSocial', 'estalecimento_noFantasia',
       'estabelecimento_municipio_codigo', 'estabelecimento_municipio_nome',
       'estabelecimento_uf', 'vacina_grupoAtendimento_codigo',
       'vacina_grupoAtendimento_nome', 'vacina_categoria_codigo',
       'vacina_categoria_nome', 'vacina_lote', 'vacina_fabricante_nome',
       'vacina_fabricante_referencia', 'vacina_dataAplicacao',
       'vacina_descricao_dose', 'vacina_codigo', 'vacina_nome',
       'sistema_origem'],
      dtype='object')

In [108]:
# 3)
vacinas_copy['vacina_categoria_nome'].value_counts()

Faixa Etária                                      51268
Outros                                             6104
Povos Indígenas                                    2049
Povos e Comunidades Tradicionais                    555
Trabalhadores de Saúde                              395
População Privada de Liberdade                       68
Comorbidades                                         49
Gestantes                                            30
Pessoas com Deficiência                               3
Pessoas de 60 anos ou mais institucionalizadas        1
Puérperas                                             1
Trabalhadores da Educação                             1
Name: vacina_categoria_nome, dtype: int64

In [116]:
# 4)
lista_estabelecimento = ['estabelecimento_valor',
       'estabelecimento_razaoSocial', 'estalecimento_noFantasia',
       'estabelecimento_municipio_codigo', 'estabelecimento_municipio_nome',
       'estabelecimento_uf']

In [118]:
# 5)
estab = vacinas_copy.loc[:,lista_estabelecimento]
estab

estabelecimento_valor  \
13                    2000571   
15                    6917291   
22                    6917291   
33                    9648968   
38                    6917291   
...                       ...   
545048                2000040   
545052                6955525   
545057                6917291   
545071                5354072   
545079                2000059   

                              estabelecimento_razaoSocial  \
13                PREFEITURA MUNICIPAL DE CRUZEIRO DO SUL   
15                     PREFEITURA MUNICIPAL DE RIO BRANCO   
22                     PREFEITURA MUNICIPAL DE RIO BRANCO   
33      DISTRITO SANITARIO ESPECIAL DE SAUDE INDIGENA ...   
38                     PREFEITURA MUNICIPAL DE RIO BRANCO   
...                                                   ...   
545048                      PREFEITURA MUNICIPAL DE FEIJO   
545052  DISTRITO SANITARIO ESPECIAL INDIGENA ALTO RIO ...   
545057                 PREFEITURA MUNICIPAL DE RIO BRANCO   
545071                     PREFEITURA MUNICIPAL DE JORDAO   
545079                      PREFEITURA MUNICIPAL DE FEIJO   

                                 estalecimento_noFantasia  \
13               UNIDADE DE SAUDE DA FAMILIA GERALDO MAIA   
15      DEPARTAMENTO DE VIGILANCIA EPIDEMIOLOGICA E AM...   
22      DEPARTAMENTO DE VIGILANCIA EPIDEMIOLOGICA E AM...   
33                         POLO INDIGENA DE MANOEL URBANO   
38      DEPARTAMENTO DE VIGILANCIA EPIDEMIOLOGICA E AM...   
...                                                   ...   
545048                  CENTRO DE SAUDE DIAMANTINO MACEDO   
545052                           POLO BASE DE MANCIO LIMA   
545057  DEPARTAMENTO DE VIGILANCIA EPIDEMIOLOGICA E AM...   
545071  UNIDADE DE SAUDE DA FAMILIA ANTONIO RODRIGUES ...   
545079                                  U S F MARIA ALICE   

        estabelecimento_municipio_codigo estabelecimento_municipio_nome  \
13                                120020                CRUZEIRO DO SUL   
15                                120040                     RIO BRANCO   
22                                120040                     RIO BRANCO   
33                                120034                  MANOEL URBANO   
38                                120040                     RIO BRANCO   
...                                  ...                            ...   
545048                            120030                          FEIJO   
545052                            120033                    MANCIO LIMA   
545057                            120040                     RIO BRANCO   
545071                            120032                         JORDAO   
545079                            120030                          FEIJO   

       estabelecimento_uf  
13                     AC  
15                     AC  
22                     AC  
33                     AC  
38                     AC  
...                   ...  
545048                 AC  
545052                 AC  
545057                 AC  
545071                 AC  
545079                 AC  

[89769 rows x 6 columns]

In [124]:
#6)
estab.drop_duplicates()

estabelecimento_valor  \
13                    2000571   
15                    6917291   
33                    9648968   
47                    6159087   
51                    2001314   
...                       ...   
163357                2001470   
241844                7123043   
256856                2000296   
400241                7058322   
406725                3042553   

                              estabelecimento_razaoSocial  \
13                PREFEITURA MUNICIPAL DE CRUZEIRO DO SUL   
15                     PREFEITURA MUNICIPAL DE RIO BRANCO   
33      DISTRITO SANITARIO ESPECIAL DE SAUDE INDIGENA ...   
47                PREFEITURA MUNICIPAL DE CRUZEIRO DO SUL   
51                 PREFEITURA MUNICIPAL DE EPITACIOLANDIA   
...                                                   ...   
163357                  PREFEITURA MUNICIPAL DE BRASILEIA   
241844                     PREFEITURA MUNICIPAL DE JORDAO   
256856                      SECRETARIA DE ESTADO DE SAUDE   
400241                  PREFEITURA MUNICIPAL DE BRASILEIA   
406725                  PREFEITURA MUNICIPAL DE BRASILEIA   

                                 estalecimento_noFantasia  \
13               UNIDADE DE SAUDE DA FAMILIA GERALDO MAIA   
15      DEPARTAMENTO DE VIGILANCIA EPIDEMIOLOGICA E AM...   
33                         POLO INDIGENA DE MANOEL URBANO   
47       SECRETARIA MUNICIPAL DE SAUDE DE CRUZEIRO DO SUL   
51                                ESF JOAO ALVES DA SILVA   
...                                                   ...   
163357                             ESF FRANCISCO DE ASSIS   
241844                        SAUDE DA FAMILIA RIBEIRINHA   
256856           HOSPITAL DA MULHER E DA CRIANCA DO JURUA   
400241                      ESF ANTONIO MONTEIRO DOS REIS   
406725                     ESF JOSE MARIA DE SOUZA SANTOS   

        estabelecimento_municipio_codigo estabelecimento_municipio_nome  \
13                                120020                CRUZEIRO DO SUL   
15                                120040                     RIO BRANCO   
33                                120034                  MANOEL URBANO   
47                                120020                CRUZEIRO DO SUL   
51                                120025                 EPITACIOLANDIA   
...                                  ...                            ...   
163357                            120010                      BRASILEIA   
241844                            120032                         JORDAO   
256856                            120020                CRUZEIRO DO SUL   
400241                            120010                      BRASILEIA   
406725                            120010                      BRASILEIA   

       estabelecimento_uf  
13                     AC  
15                     AC  
33                     AC  
47                     AC  
51                     AC  
...                   ...  
163357                 AC  
241844                 AC  
256856                 AC  
400241                 AC  
406725                 AC  

[113 rows x 6 columns]